### PR TITLE
✨ PRTL-1601 wrap project vis in HasCases

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/HasCases/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import withData from './HasCases.relay';
+export default withData(
+  p =>
+    p.viewer.explore.cases.hits.total ? React.Children.only(p.children) : null,
```
Comment (Jephuff) [inline src/packages/@ncigdc/modern_components/HasCases/index.js]: Is there an advantage to forcing the only child here, could just wrap children in a div here rather then where it's used.

Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/HasCases/index.js]: not sure what you mean. until react 16, all components are always a single child

Comment (Jephuff) [reply src/packages/@ncigdc/modern_components/HasCases/index.js]: I mean replace `React.Children.only(p.children)` with `<div>{p.children}</div>` just so who ever it is using can wrap anything with it.

Comment (alex-wilmer) [reply src/packages/@ncigdc/modern_components/HasCases/index.js]: when react 16 comes out, you can just return `this.props.children` immediately

**Comment by alex-wilmer** [comment]
@Jephuff updated

**Pull request merged**